pkgs/development/libraries/libextractor/default.nix

```diff
@@ -58,14 +59,7 @@ stdenv.mkDerivation rec {
      ++ lib.optionals gstreamerSupport
           ([ gst_all_1.gstreamer ] ++ gstPlugins gst_all_1)
      ++ lib.optionals gtkSupport [ glib gtk3 ]
-     ++ lib.optionals videoSupport [ ffmpeg_4 libmpeg2 ];
-
-  configureFlags = [
-    "--disable-ltdl-install"
-    "--with-ltdl-include=${libtool}/include"
-    "--with-ltdl-lib=${libtool.lib}/lib"
-    "--enable-xpdf"
-  ];
```

Comment:
Can you confirm why these flags were removed? They were introduced [~15 years ago](https://github.com/NixOS/nixpkgs/commit/3c1304aef23c2dbd063af2cf55f00b0e37927982).

Reply:
I did this *a while* ago and so the exact motivation has slipped my memory, but IIRC I found they were unnecessary; I believe [this upstream commit](https://git.gnunet.org/libextractor.git/commit/configure.ac?id=916cc59d27c4dedbcafc51daf42beb8d1a371416) fixed the libtool search and I believe `--disable-ltdl-install` no longer worked or at least didn’t seem like it would do anything. `--enable-xpdf` was dropped in [this upstream commit](https://git.gnunet.org/libextractor.git/commit/?id=a0655d656c37edd602dc0fd40bbb53071cbfb85b) from 2009(!) with the wonderful commit message “pdf”.